### PR TITLE
add speaker pics, partners and diversity tickets

### PR DESCRIPTION
**Pull request by jospoortvliet**
add speaker pics, partners and diversity tickets

conf_orig_256c_zopfli

Note, the color banding is due to me trying to keep the above image from being 15 mb ;-)

Signed-off-by: Jos Poortvliet <jospoortvliet@gmail.com>

### CODE REVIEW

**Comment by jospoortvliet** [comment]
TODO:
* [ ] get the design of images above and below program reviewed...
* [ ] update the LinuxMagazine logo
* [ ] get OK on diversity ticket thing

File: page-conf.php [hunk 1]
```diff
@@ -35,6 +35,14 @@
 	</div>
 </section>
 
+<section class="section--speakers" style="background: url('/wp-content/themes/next/assets/img/conference/keynotespeakers.jpg');height:500px;background-size:cover;background-position: center;">
+    <div class="container">
+        <div class="row">
+            <h1 class="text-center" style="text-shadow: 0px 0px 7px #000; color: white; margin-top:300px;">inspiring keynotes</h1>
```
Comment (MariusBluem) [inline page-conf.php]: **I**nspiring keynotes

File: page-conf.php [hunk 2]
```diff
@@ -75,6 +83,14 @@
 </div>
 </section>
 
+<section class="section--speakers" style="background: url('/wp-content/themes/next/assets/img/conference/communityspeakers.jpg');height:500px;background-size:cover;background-position: center;">
+    <div class="container">
+        <div class="row">
+            <h1 class="text-center" style="text-shadow: 0px 0px 7px #000; color: white; margin-top:300px;">share your own ideas</h1>
```
Comment (MariusBluem) [inline page-conf.php]: **S**hare your own ideas

**Comment by jancborchardt** [comment]
Regarding the diversity tickets section:
- We should add »See our event on Diversity Tickets for how to apply via mail.« – cause otherwise people who click the direct link from diversitytickets.org might be confused that they are redirected back.
- Button label not »Learn more« but »Apply for support«

**Comment by jancborchardt** [comment]
Will create the event at diversitytickets.org now with the link to https://nextcloud.com/conf/#diversity and then give you the link. ;)

**Comment by jancborchardt** [comment]
@jospoortvliet the link to the event on Diversity Tickets (once approved) will be https://diversitytickets.org/events/182

**Comment by splitt3r** [comment]
image

The buttons look a bit strange on smaller screens

**Comment by jancborchardt** [comment]
@jospoortvliet the entry is now approved and live at https://diversitytickets.org/events/182 – would be cool to get this in soon so the link actually works. :)

**Comment by jancborchardt** [comment]
@jospoortvliet any chance we can publish this and fix stuff going forward? The diversity tickets entry is already live as mentioned and there’s a tweet out too: https://twitter.com/travis_fnd/status/1006563963008638976

**Comment by jospoortvliet** [comment]
On it ;-)

File: page-conf.php (hunk 1 above)
Comment (jospoortvliet) [reply page-conf.php]: :+1: done

File: page-conf.php (hunk 2 above)
Comment (jospoortvliet) [reply page-conf.php]: :+1: done

**Comment by jancborchardt** [comment]
@jospoortvliet just tested, it links to https://nextcloud.com/conf/#diversity – the subsection. I think that's fine though, as then you are led to the detail page.

The problem is only that on the list of events at diversitytickets, the buttons directly link out instead of showing the detail view of the event.

It'll be fine, we can always iterate :)

**Pull request merged**